### PR TITLE
Replaced PosixPath with PurePosixPath for Windows compatibility.

### PR DESCRIPTION
**Pull request by ranguwud**

### CODE REVIEW

File: pypga/core/interface/remote/server.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __init__(self, host, port=2222, delay=0.05, bitstreamfile=None, start=True):
             hostname=host,
             sshport=22,
             user="root",
-            password="root",
+            password="qlibri2000xxx",
```
Comment (lneuhaus) [inline pypga/core/interface/remote/server.py:33]:
```suggestion
            password="root",
```

Comment (lneuhaus) [reply pypga/core/interface/remote/server.py:33]: Should we allow setting this via an environment variable `REDPITAYA_PASSWORD`?

Comment (ranguwud) [reply pypga/core/interface/remote/server.py:33]: Oops, I didn't realize that I had staged this change. Yes, definitely. In general, I'm not a huge fan of environment variables and would rather allow setting such config through a global variable from code. But I believe that both can be combined and an environment variable is probably a good start.

Comment (ranguwud) [reply pypga/core/interface/remote/server.py]: I have now added a new commit, which allows setting user and password by environment variables as well as from code before calling `run()`. I'm not super happy with the way it is implemented, maybe you have some suggestion.

Comment (lneuhaus) [reply pypga/core/interface/remote/server.py]: We had already a file for settings/config: https://github.com/pypga/pypga/blob/main/pypga/core/settings.py. Maybe we can extend that one? Let me know if you need help with that.

**Review by josselex — APPROVED** [review]

**Pull request merged**